Patches/Caffe/WindowsDownloadPrebuiltDependencies.cmake

```diff
@@ -0,0 +1,182 @@
+set(DEPENDENCIES_URL_1800_27 "https://github.com/willyd/caffe-builder/releases/download/v1.1.0/libraries_v120_x64_py27_1.1.0.tar.bz2")
```

Comment:
I would credit the location you got this from if you didn't write part of it (e.g. if it was from the caffe windows branch) at the top